packages/alfa-dom/src/get-input-type.ts

```diff
@@ -22,7 +22,8 @@ export type InputType =
   | "submit"
   | "image"
   | "reset"
-  | "button";
+  | "button"
+  | "week";
```

Comment:
I'd order these the same way they're ordered in the spec, i.e. move `"week"` up.

Reply:
Week is the last in the spec as far as I can see.

Reply:
It comes after `"month"` in the overview:

![image](https://user-images.githubusercontent.com/1174718/41462169-3a4616cc-7092-11e8-895c-be13c94393b8.png)